File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        s = set(nums)
        # print()
        # print(len(s))
        return len(s) != len(nums)
            # return False
        # else:
            # return True

File: 0217-contains-duplicate/0217-contains-duplicate.java
// import java.util.*;
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> set = new HashSet<>();
        for(int i: nums){
            if(!set.contains(i))
                set.add(i);
            else
                return true;
        }
        return false;
    }
}